### DISCUSSION TITLE
sensor.oig_xxxxxxxxx_invertor_prms_to_grid state N/A

### DISCUSSION THREAD

**Discussion by jiripatka** [psimsa/oig_cloud#70] [Q&A]
Hello, since 25.9.  02:09 my Battery box does not send entity status: Overflows on network = unknown. Do you have the same experience ? I have a condition in automation controlling overflows by current spot price that asks for current state of sensor.oig_xxxxxxx_invertor_prms_to_grid. Do you also use this entity to disable overflows ?

**Comment by PlutoWRC** [comment]
Zdravim. Tato entita nyní bohužel nefunguje. Celou tuto záležitost řešíme na FB: https://www.facebook.com/share/p/z2GQ9qA6S3B1df2C/
